### PR TITLE
github_webhook: Don't include secret in the config if it's absent

### PR DESCRIPTION
**Pull request by dima1206**
##### SUMMARY
Currently if you don't have a secret for the hook, the module will always report `changed` status even after multiple runs. This change adds secret to the config only if user specifies it.

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
github_webhook

##### ADDITIONAL INFORMATION
I've tested the change locally:
- if user doesn't provide secret - it doesn't report `changed` state anymore on the second run
- if user provides empty secret - it doesn't report `changed` state anymore on the second run
- if there was a secret before, and user doesn't provide a secret in the task - it will remove the secret (same as before this change)

### CODE REVIEW

**Comment by dima1206** [comment]
What is `ansible/third-party-check` and why does it fail?

File: plugins/modules/github_webhook.py [hunk 1]
```diff
@@ -154,13 +154,18 @@
...
-    return {
+    hook_config = {
         "url": module.params["url"],
         "content_type": module.params["content_type"],
-        "secret": module.params.get("secret"),
         "insecure_ssl": "1" if module.params["insecure_ssl"] else "0"
     }
 
+    secret = module.params.get("secret")
+    if secret is not None and len(secret) > 0:
```
Comment (felixfontein) [inline plugins/modules/github_webhook.py:164]: You can also shorten this to:
```suggestion
    if secret:
```

**Review by felixfontein — COMMENTED** [review]
Thanks for your contribution! A first quick comment:

**Comment by felixfontein** [comment]
> What is `ansible/third-party-check` and why does it fail?

Simply ignore that. It currently does that a lot.

**Comment by felixfontein** [comment]
Looks OK to me. Will merge this weekend if nobody objects.

**Comment by felixfontein** [comment]
@dima1206 thank you very much for your contribution!
